iesurveychecks/2-set-up-hfc-dashboard.md

```diff
@@ -0,0 +1,83 @@
+
+  5. <b>Data Quality (by Enumerator, recent data only)</b>:  Considering that during later stages of data collection, recent trends might be neglected and unnoticeable due to the large volume of existing data, therefore, it is important to create a separate data quality sheet just for data submitted over the past 7 days. This also allows us to capture if enumerators have rectified any early mistakes that we catch. The most easiest way to do this is
+    - Duplicate the Data Quality Full Data sheet
+    - Add a filter argument to each PivotTable you’ve created and under “filter by condition”, select “Date is within past week”
+      - Note: you need to first export the “SubmissionDate” variable to your Google Sheet dashboard and then convert it to datetime format
+      <img src="https://github.com/dime-worldbank/iesurveykit/blob/initial-update/iesurveychecks/img/image6.png"><!--- Image is read from master branch or use full URL-->
+
+  6. <b>Data Quality (by Enumerator or by a particular date)</b>:  For the same reasons as mentioned above, it is necessary to have the option to filter for specific enumerators or specific dates. The easiest way to do this is
+      - Duplicate the Data Quality Full Data sheet twice (one for filtering for enumerator, one for date). Remove any tables which are not relevant for these tabs like those tabulating responses by date and by enumerator in a single table.
+      - Go to “Data” in the Google Sheets Toolbar and select “Add a slicer”.
+      - Then select the sheet that your pivot tables are pulling data from (in our sample dashboard, this is the “trackdata” sheet) and select the entire dataset.  
+      - Select the variable you want to filter by. In our sample dashboard, we use enumerator name to filter under (`Enum_Wise`) and date in (`Date_Wise`). Ensure that the box is checked to apply this slicer to all pivot tables on that sheet.
+      <img src="https://github.com/dime-worldbank/iesurveykit/blob/initial-update/iesurveychecks/img/image4.png"><!--- Image is read from master branch or use full URL-->
+
+      - `Enum_Wise` demonstrates how to view performance of one or more enumerators over time. Similarly, `Date_Wise` allows us to view performance on a particular day or selected dates by all enumerators. This is relevant if there are some dates which are crucial to monitor, or if a single or group of enumerators’ performance needs to be observed. The filters can be updated even without access to edit the dashboard.
+
+Details on how to use the Google sheets functions used in the real time HFC dashboard can be found [here](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/iesurveychecks/google-functions.md).
+
+
+
+
+The template real time data quality checks dashboard uses conditional formatting to highlight different types of Completion and Survey Outcomes.
```

Comment:
would be helpful to have a header here (this section is long and this seems like a natural break)